thriftclient/testing.go

```diff
@@ -0,0 +1,103 @@
+package thriftclient
```

Comment:
Should this actually go in a `thrifttesting` subpackage or something like that, similar to the `httptest` subpackage in `net/http`?  Or is that overkill?

Reply:
Since this package is small enough this is fine.